internal/api/langs/lang_jssdk.go

```diff
@@ -0,0 +1,38 @@
+//go:build jssdk
+
+package langs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/matrix-org/complement-crypto/internal/api"
+	"github.com/matrix-org/complement-crypto/internal/api/js"
+	"github.com/matrix-org/complement/ct"
+	"github.com/matrix-org/complement/must"
+)
+
+// Can't use tag name 'js' as that is used already for wasm bindings...
```

Comment:
Is this comment in the wrong place?

Reply:
No it's in the right place, ish. It's the right file, and can't be next to the go:build directive, so here is probably the best place for it.